testing/upload_validation.ts

```diff
@@ -2269,6 +2269,11 @@ ${endpointKey ? 'endpointKey is set' : `requiresEndpointUrl is ${requiresEndpoin
             skillName: z.string(),
           })
           .optional(),
+        defaultChat: z
+          .object({
+            skillName: z.string(),
+          })
```

Comment:
Can you split this out into a shared validator, and that uses `zodCompleteObject<SkillEntrypointConfig>` so that we get TS enforcement that new fields in the config object require an update to the validator.